tools/alpine/push-manifest.sh

```diff
@@ -46,7 +46,7 @@ EOF
 # we need them for notary on all platforms.
 case $(uname -s) in
     Darwin)
-        CRED=$(echo "https://index.docker.io/v1/" | /Applications/Docker.app/Contents/Resources/bin/docker-credential-osxkeychain.bin get)
+        CRED=$(echo "https://index.docker.io/v1/" | /Applications/Docker.app/Contents/Resources/bin/docker-credential-osxkeychain get)
```

Comment:
is it worth supporting both names for a while?

Reply:
Probably. Pushed  an update